api-lib/neynar/findOrCreate.ts

```diff
@@ -154,6 +155,11 @@ const createProfile = async (
     console.log('Creating new profile with name', name);
   }
 
+  // if we have an avatar_url, upload it to cloudflare for our use
+  const avatar = avatar_url
+    ? await uploadURLToCloudflare(avatar_url)
```

Comment:
this is gonna error in test/dev if you dont have a cloduflare token or valid keys?

Reply:
fixed that thanks